### PR TITLE
Combined all themes in single template file.

### PR DESCRIPTION
**Pull request by Scopestyle**
All themes are now combined in a single template "modern".

Kind regards,

Erik

### CODE REVIEW

**Comment by thc202** [comment]
The commit needs to be signed off: https://github.com/zaproxy/zap-extensions/pull/2961/checks?check_run_id=2716192832

**Comment by Scopestyle** [comment]
@thc202 Commit is now signed off. 👍

**Comment by kingthorin** [comment]
Your git config doesn't seem to match your GitHub account info.
wk.local?
https://github.com/zaproxy/zap-extensions/commit/b22cf70a7f6bca2d9a9be821efc5a8afdbf425a6.patch

**Comment by Scopestyle** [comment]
@kingthorin I'm not an avid GIT user, is this really a problem? What do I have to do to get this merged?

**Comment by kingthorin** [comment]
No no, if you don't care it doesn't matter.

File: addOns/reports/src/main/zapHomeFiles/reports/modern/report.html [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+										th:text="${alert.userObject.pluginId}" href="plugin.html">Plugin
+											Id</a></td>
+								</tr>
+							</table>
+						</div>
+						<div class="spacer"></div>
+					</th:block>
+				</th:block>
+	
+				<footer>Report by <a href="https://www.zaproxy.org/" target="_blank">ZAP</a></footer>
```
Comment (psiinon) [inline addOns/reports/src/main/zapHomeFiles/reports/modern/report.html:495]: This should be i18n'd

**Review by psiinon — CHANGES_REQUESTED** [review]
Just spotted pone string that needs to be internationalized, other than that its looking very good to me :)

**Comment by psiinon** [comment]
It will require a spotless apply in order to pass the build, eg `./gradlew :addOns:reports:spotlessApply`

**Comment by psiinon** [comment]
Replaced by #2976

**Comment by kingthorin** [comment]
@Scopestyle #2976 has been merged now, how would you like to appear in the credits? (Have a look at the current entries and let us know)

https://www.zaproxy.org/docs/desktop/credits/#zap-extended-team

**Comment by Scopestyle** [comment]
@kingthorin Feel free to add my name and Twitter Handle: Erik de Kuijper ([@edkpr](https://www.twitter.com/edkpr))
Thank you!

**Pull request closed without merging**